api/src/main/java/module-info.java

```diff
@@ -0,0 +1,6 @@
+module jakarta.servlet {
+    exports jakarta.servlet;
+    exports jakarta.servlet.annotation;
+    exports jakarta.servlet.descriptor;
+    exports jakarta.servlet.http;
```

Comment:
Jetty has been adding JPMS to our re-bundling of 4.0 API and we have included an export for jakarta.servlet.resources which includes all the XSDs and DTDs.    I believe tomcat also bundles the resources the same way.

I don't think we really want to have another JPMS module involved that has the XSDs and DTDs... if so should this module depend on that?

Reply:
Probably. I'm not sure how to proceed right now. 
Could you please give me a hint in what I should look next?

Reply:
yeah, it's not something that you can do on your own here.  It is a policy decision that has to be made at least by the project, but more likely by the EE community as a whole.   So I think it's a bit of a road block, but unfortunately due to the nature of JPMS modules, we really don't want to get it wrong and then try to modify later.  The first published non auto module name needs to be pretty correct in content and intent.    

@stuartwdouglas @markt-asf @bshannon @dblevins any thoughts on this one?

Reply:
You shouldn't export resources that aren't defined by the spec.
The spec doesn't define a resource location for the schemas.